core/src/test/scala/unit/kafka/admin/ResetConsumerGroupOffsetTest.scala

```diff
@@ -334,8 +335,14 @@ class ResetConsumerGroupOffsetTest extends ConsumerGroupCommandTest {
     adminZkClient.deleteTopic(topic)
   }
 
+  private def produceMessages(numMessages: Int): Unit = {
+    val records = (0 until numMessages).map(_ => new ProducerRecord[Array[Byte], Array[Byte]](topic,
+      new Array[Byte](100 * 1000)))
+    TestUtils.produceMessages(servers, records, acks = 1)
+  }
+
   private def produceConsumeAndShutdown(totalMessages: Int, numConsumers: Int = 1, topic: String = topic) {
-    TestUtils.produceMessages(servers, topic, totalMessages, acks = 1, 100 * 1000)
+    produceMessages(totalMessages)
```

Comment:
we are not passing topic to produceMessages. This is causing some tests to fail.

Reply:
We don't need to pass the topic since the instance method does that. I'll check the failures.

Reply:
You are right actually, this class is written in a confusing way.

Reply:
Fixed it.